OrbitService/OrbitGrpcServer.cpp

```diff
@@ -29,9 +30,10 @@ class OrbitGrpcServerImpl final : public OrbitGrpcServer {
   void Wait() override;
 
  private:
-  CrashServiceImpl crash_service_;
+  CaptureServiceImpl capture_service_;
   ProcessServiceImpl process_service_;
   FramePointerValidatorServiceImpl frame_pointer_validator_service_;
+  CrashServiceImpl crash_service_;
```

Comment:
nit: | sort?

Reply:
I was thinking of alphabetical..

Reply:
I prefer consistency with other files (services.proto) to an arbitrary order (even if it's alphabetical). Sure, I can reorder services.proto as well, but the current order makes sense to me.